tests/unit/src/main/scala/tests/MetalsTestEnrichments.scala

```diff
@@ -60,13 +60,14 @@ object MetalsTestEnrichments {
         val input = source.toInput
         val symbols = ArrayBuffer.empty[WorkspaceSymbolInformation]
         val methodSymbols = ArrayBuffer.empty[WorkspaceSymbolInformation]
-        SemanticdbDefinition.foreach(input, dialect) {
+        SemanticdbDefinition.foreach(input, dialect, includeMembers = true) {
```

Comment:
```suggestion
        val includeMembers = wsp.workspace.extension != "zip" && wsp.workspace.extension != "jar" 
        SemanticdbDefinition.foreach(input, dialect, includeMembers) {
```
seems `workspace` is the actual sources in some of the tests.